docs/upgrading.md

```diff
@@ -7,6 +7,12 @@ Below we describe the additional manual steps required to upgrade between differ
 
 # Open AEA
 
+## `v1.7.0` to `1.8.0`
+
+No backwards incompatible changes.
```

Comment:
That's not true this time? Are all ipfs hashing related changes (wrapper & public id) fully backwards compatible.

In any case, add a para on the change in the hashing and public id structure.